README.md

```diff
@@ -55,6 +59,19 @@ If linting reveals some error, they can be fixed with
 
     $ ./build.js lintfix
 
+Although just doing the linting will reveal the rules, there are a couple of
+sources for this configuration:
+
+* [`.editorconfig`](.editorconfig) is a standard way of conveying to all kinds
+  of editors and IDEs general configuration values such as how many spaces using
+  in indenting, and max line length. This is going to be checked any way, but
+  it's a way of enforcing it at the editor level.
+* [`.eslintrc.cjs`](.eslintrc.cjs) describes other globals and settings that
+  will be enforced when running linting. In general, we will follow [standard
+  rules](https://eslint.org/docs/rules/), with this module just turning off some
+  of them. Please familiarize yourself with them, because the linter is
+  unforgiving.
```

Comment:
I was aiming to keep the top level README file short - any chance we can shorten this part? Generally, it's pretty easy to check that we're using ESLint, and it's also then pretty easy to figure out what files it picks up rules from. Not super opposed to listing them here, but not super happy with bloating the README either.